Makefile

```diff
@@ -58,9 +58,12 @@ coverage: clean  ## Run python tests with coverage
 	nosetests --with-coverage
 
 .PHONY: scan
-scan: ./build/lambda.zip ## Run scan function locally
-	scripts/run-scan-lambda $(TEST_BUCKET) $(TEST_KEY)
+scan:
+	sam local generate-event s3 [put/delete] \
```

Comment:
Not sure why this is [put/delete], rather than just put, but I can see from the docs that this will generate a sample payload for adding a new object to an S3 bucket, so 👍